### PR TITLE
fixing documentation for tf.batch_to_space

### PR DESCRIPTION
**Pull request by vishalsubbiah**
issue #33354
each step is not separated correctly : https://www.tensorflow.org/api_docs/python/tf/batch_to_space

replacing #33351

### CODE REVIEW

**Comment by vishalsubbiah** [comment]
cc @mihaimaruseac

**Review by mihaimaruseac — DISMISSED** [review]
Might need to run lint on the code to check formatting, but looks good

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by vishalsubbiah** [comment]
@rthadur can i redo a certain check? the error for Ubuntu Sanity is 
```
ERROR: /tmpfs/src/github/tensorflow/tensorflow/tools/ci_build/BUILD:3:1:  failed: No usable spawn strategy found for spawn with mnemonic TestRunner.  Your --spawn_strategy, --genrule_strategy or --strategy flags are probably too strict. Visit https://github.com/bazelbuild/bazel/issues/7480 for migration advice
```
This shouldn't be because of my change

File: tensorflow/python/ops/array_ops.py [hunk 1]
```diff
@@ -3432,16 +3432,18 @@ def batch_to_space_v2(input, block_shape, crops, name=None):  # pylint: disable=
...
+        input_shape[N-1]]  
+      2. Permute dimensions of `reshaped` to produce `permuted` of shape 
+         [batch / prod(block_shape),  input_shape[1], block_shape[0], ..., 
+         input_shape[M], block_shape[M-1], input_shape[M+1],
+        ..., input_shape[N-1]]  
+      3. Reshape `permuted` to produce `reshaped_permuted` of shape 
+         [batch / prod(block_shape), input_shape[1] * block_shape[0], ..., 
+         input_shape[M] * block_shape[M-1], input_shape[M+1], ..., 
+         input_shape[N-1]]  
+      4. Crop the start and end of dimensions `[1, ..., M]` of `reshaped_permuted`
```
Comment (rthadur) [inline tensorflow/python/ops/array_ops.py]: `FAIL: Found 1 non-whitelisted pylint errors:
tensorflow/python/ops/array_ops.py:3445: [C0301(line-too-long), ] Line too long (82/80)`
can you please fix this sanity error

**Review by rthadur — CHANGES_REQUESTED** [review]

**Review by rthadur — APPROVED** [review]

**Pull request merged**